streamlit_folium/__init__.py

```diff
@@ -5,10 +5,11 @@
 import re
 import warnings
 from textwrap import dedent
-from typing import Iterable
+from typing import Iterable, List
```

Comment:
**nitpick:** Because of line 1, you can just use `list[str]` instead of importing `List`

Reply:
Update committed!